diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index 20f41ea..fbe6220 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -131,6 +131,7 @@ repositories {
 }
 
 android {
+    namespace "com.swmansion.gesturehandler"
     compileSdkVersion safeExtGet("compileSdkVersion", 28)
 
     // Used to override the NDK path/version on internal CI or by allowing
@@ -175,6 +176,10 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    kotlinOptions {
+        jvmTarget = "1.8" // Use a valid target like 1.8 or 11
+    }
+
     if (isNewArchitectureEnabled()) {
         externalNativeBuild {
             cmake {
@@ -211,6 +216,10 @@ android {
             }
         }
     }
+
+    buildFeatures {
+        buildConfig true
+    }
 }
 
 def kotlin_version = safeExtGet('kotlinVersion', project.properties['RNGH_kotlinVersion'])
